diff --git a/node_modules/@arcana/auth-react-native/index.js b/node_modules/@arcana/auth-react-native/index.js
index 400d92f..9787af0 100644
--- a/node_modules/@arcana/auth-react-native/index.js
+++ b/node_modules/@arcana/auth-react-native/index.js
@@ -4,6 +4,7 @@ import {
   StyleSheet,
   Keyboard,
   View,
+  Platform,
 } from "react-native";
 import { randomNumber, createLoginUrl } from "./utils";
 import React, { useRef, useState, useEffect } from "react";
@@ -80,6 +81,10 @@ const Auth = React.forwardRef((props, ref) => {
     return loginState;
   };
 
+  const getWalletVisibility = () => {
+    return visible
+  }
+
   const sendTransaction = (params) => {
     assertLoggedIn();
     return new Promise((resolve, reject) => {
@@ -239,15 +244,17 @@ const Auth = React.forwardRef((props, ref) => {
     sendTransaction,
     getAccount,
     getBalance,
+    getWalletVisibility
   }));
 
   const visibilityCss = {
     ...styles.modalContainer,
     maxHeight: "100%",
     height: deviceHeight - keyboardHeight,
-    bottom: 0,
+    //bottom: 0,
     // top: 0,
-    translateY: bottom,
+    transform:[{translateY:bottom}]
+    //translateY: bottom,
   };
   
   return (
@@ -340,7 +347,7 @@ const styles = StyleSheet.create({
     left: 0,
     right: 0,
     bottom: 0,
-    backgroundColor: "transparent",
+    backgroundColor:"transparent",
   },
   modalContent: {
     height: "100%",
